structuredefinitions/Extension-England-TypedPeriod.xml

```diff
@@ -1,11 +1,11 @@
 <StructureDefinition xmlns="http://hl7.org/fhir">
   <id value="Extension-England-TypedPeriod" />
   <url value="https://fhir.nhs.uk/England/StructureDefinition/Extension-England-TypedPeriod" />
-  <version value="1.0.0" />
+  <version value="2.0.0" />
```

Comment:
Can we have this as 1.1.0 as adding a new code has been minor for both codesystem and valueset, so not sure why this would be a major change here